.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cSpell.words": ["turbopack", "vercel"],
+  "cSpell.words": ["atproto", "turbopack", "vercel"],
```

Comment:
this could be in your user workspace settings since we don't recommend this extension